Docstrings -> Google style for mkdocs render (#14)
datajoint/workflow-deeplabcut: README.md

# DataJoint Workflow - DeepLabCut

> This repository demonstrates the functionality of the 
> DataJoint Element for markerless pose estimation with
> [DeepLabCut](https://www.deeplabcut.org/).  DataJoint Elements collectively standardize
> and automate data collection and analysis for neuroscience experiments.  Each Element is
> a modular pipeline for data storage and processing with corresponding database
> tables that can be combined with other Elements to assemble a fully functional pipeline.

Reviewer: ```suggestion
The DataJoint Workflow for DeepLabCut combines multiple DataJoint Elements for markerless pose estimation with
[DeepLabCut](https://www.deeplabcut.org/).  DataJoint Elements collectively standardize
and automate data collection and analysis for neuroscience experiments.  Each Element is
a modular pipeline for data storage and processing with corresponding database
tables that can be combined with other Elements to assemble a fully functional pipeline.
```

> Installation and usage instructions, including those featured in this repository,
> can be found at the 
> [Element documentation](datajoint.com/docs/elements/element-deeplabcut).

Reviewer: ```suggestion
Installation and usage instructions can be found at the 
[Element documentation](datajoint.com/docs/elements/element-deeplabcut).
```
